src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color,int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square " + "area: " + area() + " sq. units"
```

Comment:
Use name() for getting String representation of enum constants. If 'color' is an enum, you should use color.name() instead of directly referring to 'color'.